ipyleaflet/projections.py

```diff
@@ -25,44 +25,146 @@
         name='Simple',
         custom=False
     ),
-    EPSG3413=dict(
-        name='EPSG3413',
-        custom=True,
-        proj4def="""+proj=stere +lat_0=90 +lat_ts=70 +lon_0=-45 +k=1 +x_0=0 +y_0=0
-                 +ellps=WGS84 +datum=WGS84 +units=m +no_defs""",
-        origin=[-4194304, 4194304],
-        resolutions=[
-            16384.0,
-            8192.0,
-            4096.0,
-            2048.0,
-            1024.0,
-            512.0,
-            256.0
-        ],
-        bounds=[
-            [-4194304, -4194304],
-            [4194304, 4194304]
-        ]
+    EPSG3413=Bunch(
+        NASAGIBS=dict(
```

Comment:
`EPSG:3413` means it goes with `NASAGIBS`, or is it independent?

Reply:
I struggled here since this would break backwards compatibility. There are different options for `EPSG:3031` in terms of bounds and number of zoom levels. I figured there was similar for `EPSG:3413`. I felt like having another level of attributes was cleaner than having `EPSG3031_basemap` or `EPSG3031_NASAGIBS`. Suggestions?